tests/generic_relations/tests.py

```diff
@@ -499,6 +499,53 @@ def test_assign_content_object_in_init(self):
         tag = TaggedItem(content_object=spinach)
         self.assertEqual(tag.content_object, spinach)
 
+    def test_create_after_prefetch(self):
+        platypus = Animal.objects.prefetch_related('tags').get(pk=self.platypus.pk)
+        self.assertQuerysetEqual(platypus.tags.all(), [])
+        platypus.tags.create(tag='weird')
+        self.assertQuerysetEqual(platypus.tags.all(), ["<TaggedItem: weird>"])
```

Comment:
We're generally preferring `assertCountEqual()` and `assertSequenceEqual()` over `assertQuerysetEqual()` to remove the need to put a bunch of `__str__()` strings.